src/main/java/io/numaproj/kafka/producer/KafkaSinker.java

```diff
@@ -46,6 +47,13 @@ public KafkaSinker(
     this.userConfig = userConfig;
     this.producer = producer;
     this.schemaRegistry = schemaRegistry;
+    this.schema = schemaRegistry.getAvroSchema(this.userConfig.getTopicName());
+    if (schema == null) {
```

Comment:
What if there is no schema associated to a topic? Can we turn on/off integration with schema registry?

Reply:
Yes, I am working on it as part of https://github.com/numaproj-contrib/kafka-java/issues/5